eth/block_test.go

```diff
@@ -570,3 +570,34 @@ func TestBlock_BaikalBlock(t *testing.T) {
 	require.NoError(t, err)
 	require.JSONEq(t, string(j), string(j2))
 }
+
+func TestBlock_ZhejiangBlock(t *testing.T) {
```

Comment:
Could we add a comment that this unit test tests our `json.Marshal/Unmarshal` logic?